tests/functional/aws-node-sdk/test/object/putObjectLegalHold.js

```diff
@@ -18,17 +18,26 @@ const mockLegalHold = {
     on: { Status: 'ON' },
 };
 
-function createLegalHoldParams(bucket, key, status) {
+function createLegalHoldParams(bucket, key, status, versionId) {
+    const params = { Bucket: bucket, Key: key };
     if (!status) {
-        return { Bucket: bucket, Key: key };
+        return params;
     }
-    return {
-        Bucket: bucket,
-        Key: key,
+    if (!versionId) {
+        Object.assign(params, {
+            VersionId: versionId,
```

Comment:
Isn't the logic reversed here? if `versionId` is false-ish it's likely undefined.

Reply:
I mean the logic of the `if`

Reply:
it is -- supposed to be if `versionId` not `!versionId` just fixed it should be good now.